src/angleaxis_types.jl

```diff
@@ -79,10 +79,11 @@ end
 end
 
 @inline function Base.convert(::Type{AA}, q::Quat) where AA <: AngleAxis
-    # TODO: consider how to deal with derivative near theta = 0
-    s = sqrt(q.x*q.x + q.y*q.y + q.z*q.z)
-    theta =  2 * atan(s, q.w)
-    return s > 0 ? AA(theta, q.x / s, q.y / s, q.z / s, false) : AA(theta, one(theta), zero(theta), zero(theta), false)
+    s2 = q.x * q.x + q.y * q.y + q.z * q.z
+    cos_t2 = sqrt(s2)
+    theta = 2 * atan(cos_t2 / abs(q.w))
+    sc = ifelse(cos_t2 > 0, promote(copysign(1 / cos_t2, q.w), 2)...) # N.B. the 2 "should" match the derivative as cos_t2 -> 0
+    return AA(theta, sc * q.x, sc * q.y, sc * q.z)
```

Comment:
Made this more similar to `Base.convert(::Type{RV}, q::Quat)`, which I believe should also resolve the TODO regarding derivatives near theta = 0.